src/main/kotlin/com/github/ast/parser/KParserImpl.kt

```diff
@@ -445,11 +446,9 @@ open class KParserImpl(
     inline fun <reified T : Enum<T>> addControls(control: UINode, className: String) {
         enumValues<T>().forEach {
             if (control.nodeType.toLowerCase() == (it.name).toLowerCase()) {
-                if (!detectedUIControls.containsKey(className)) {
-                    val controlCollection = ArrayList<UINode>()
-                    detectedUIControls[className] = controlCollection
-                }
-                detectedUIControls[className]?.add(control)
+                detectedUIControls.getOrPutNullable(className){
+                    ArrayList()
```

Comment:
is `getOrPut`/`getOrPutNullable` new in Kotlin?

Reply:
Add a space to `detectedUIControls.getOrPutNullable(className) {` (and same for the file below) but I'll approve!